src/components/TopBar/CallTime.vue

```diff
@@ -20,17 +20,17 @@
 
 <template>
 	<NcPopover class="call-time"
-		close-after-click="true"
 		:menu-title="callTime"
 		:shown.sync="showPopover"
+		:focus-trap=" isShowRecordingStatus "
 		:triggers="[]"
 		:container="container">
 		<template #trigger>
-			<NcButton :disabled="!isShowRecordingStatus || !isModerator"
+			<NcButton :disabled="(!isShowRecordingStatus || !isModerator ) && !isCallDurationHintShown"
```

Comment:
```suggestion
			<NcButton :disabled="isButtonDisabled"
```
let's move it to computed as well